src/main/resources/templates/index.html

```diff
@@ -48,6 +50,9 @@
                         </button>
                     </form>
                 </td>
+                <td>
+                    <p th:text="${survey.isOpen()? 'Open' : 'Closed'}"></p>
```

Comment:
you also should make it so that the index page won't link to the edit page if the survey is open
and won't link to the fill out page if it is closed